### PR TITLE
Fix missing dependency and add license files in Dockerfile

### PR DESCRIPTION
**Pull request by PragmaTwice**

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]

**Review by git-hulk — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -38,5 +40,8 @@ COPY ./kvrocks.conf  ./conf/
 RUN sed -i -e 's%dir /tmp/kvrocks%dir /var/lib/kvrocks%g' ./conf/kvrocks.conf
 VOLUME /var/lib/kvrocks
 
+COPY ./LICENSE ./NOTICE ./DISCLAIMER .
+COPY ./licenses ./licenses
```
Comment (tisonkun) [inline Dockerfile]: Step 16/19 : COPY ./LICENSE ./NOTICE ./DISCLAIMER .
When using COPY with more than one source file, the destination must be a directory and end with a /

**Review by tisonkun — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (tisonkun) [reply Dockerfile]: Please verify locally first :)

Comment (PragmaTwice) [reply Dockerfile]: It is weird. My local docker does not have this error : (

File: Dockerfile [hunk 2]
```diff
@@ -38,5 +40,8 @@ COPY ./kvrocks.conf  ./conf/
 RUN sed -i -e 's%dir /tmp/kvrocks%dir /var/lib/kvrocks%g' ./conf/kvrocks.conf
 VOLUME /var/lib/kvrocks
 
+COPY ./LICENSE ./NOTICE ./DISCLAIMER .
```
Comment (PragmaTwice) [inline Dockerfile]:
```suggestion
COPY ./LICENSE ./NOTICE ./DISCLAIMER ./
```

File: Dockerfile (hunk 1 above)
Comment (PragmaTwice) [reply Dockerfile]: done

**Review by tisonkun — APPROVED** [review]

**Pull request merged**